reconcile/utils/saasherder.py

```diff
@@ -196,6 +197,38 @@ def _collect_parameters(container):
                 parameters[k] = json.dumps(v)
         return parameters
 
+    @staticmethod
+    def _get_file_contents_github(repo, path, commit_sha):
+        try:
+            f = repo.get_contents(path, commit_sha)
+            return f.decoded_content
+        except GithubException as e:
+            # slightly copied with love from
+            # https://github.com/PyGithub/PyGithub/issues/661
+            errors = e.data['errors']
+            # example errors dict that we are looking for
+            # {
+            #    'message': '<text>',
+            #    'errors': [{
+            #                  'resource': 'Blob',
+            #                  'field': 'data',
+            #                  'code': 'too_large'
+            #               }],
+            #    'documentation_url': '<url>'
+            # }
+            for error in errors:
+                if error['code'] == 'too_large':
```

Comment:
Where does this code come from? Do we know it is not going to change?

Reply:
it comes from the github api (couldn't find the exact location). we are indeed not sure, but if it changes we will know right away since this code will stop functioning. can you think of a better approach?

Reply:
I was thinking if we actually need to check for the `too_large`, but since you're raising anyway I guess we will soon know.

I've found tons of mentions to the `too_large` string, I don't think it will easily change.